core/jobs/job_utils_test.py

```diff
@@ -191,6 +191,14 @@ def test_get_model_from_beam_entity(self) -> None:
             FooModel(id='abc', project=feconf.OPPIA_PROJECT_ID, prop='123'),
             job_utils.get_ndb_model_from_beam_entity(beam_entity))
 
+    def test_get_beam_key_from_ndb_key(self):
+        beam_key = beam_datastore_types.Key(
+            ('FooModel', 'abc'), project=feconf.OPPIA_PROJECT_ID,
+                namespace=self.namespace)
+
+        ndb_key = datastore_services.Key._from_ds_key(beam_key.to_client_key())  # pylint: disable=protected-access
```

Comment:
Explain, in a comment, why we have no choice but to use the private method. (In general, if you use a pylint pragma, you should make this clear.)